ee/clickhouse/queries/funnels/funnel_persons.py

```diff
@@ -1,15 +1,31 @@
 from ee.clickhouse.queries.funnels.base import ClickhouseFunnelBase
-from ee.clickhouse.sql.funnels.funnel import FUNNEL_PERSONS_SQL
+from ee.clickhouse.sql.funnels.funnel import FUNNEL_PERSONS_BY_STEP_SQL
 from posthog.models import Person
 
 
 class ClickhouseFunnelPersons(ClickhouseFunnelBase):
     def get_query(self, format_properties):
```

Comment:
Small refactoring note for the future (not necessary for now, but for when we move `FunnelNew`->`Funnel`):

It seems to me like `format_properties` are not generic enough anymore to warrant a place in the basic `get_query` signature? maybe it's worth every query generating its own properties that it needs.

For example, here it's a bit confusing that `offset` is missing, but turns out it comes from `format_properties`, and is literally equal to `self._filters.offset` - which seems to be the only thing being used